lib/Hey/Timer.rakumod

```diff
@@ -102,15 +102,16 @@ our sub display-timers-summary-as-table(@timer_hashes, $title) is export {
 	my $total_seconds = 0;
 	my %project_times = ();
 	for @timer_hashes -> %timer_hash {
-		if %timer_hash<ended_at> ~~ Int {
-			my $timer_seconds = (%timer_hash<ended_at> - %timer_hash<started_at>);
-			$total_seconds += $timer_seconds;
-			for %timer_hash<projects>.map({$_<name>}) -> $project {
-				%project_times.EXISTS-KEY($project)
-					?? (%project_times{$project} += $timer_seconds)
-					!! (%project_times{$project} = $timer_seconds);
-			}
-		}
+		# Running timers don't have an end-time; treat them as 0-length
+		my $timer_seconds = %timer_hash<ended_at> ~~ Int:D
```

Comment:
Yeah, it might as well be.  (Since `Int:U` is falsy, the only time that would give a different answer is if  the final end time is `0`, which shouldn't happen and would probably cause other issues.  But still, it's worth adding; thanks).